libminifi/include/core/ProcessContext.h

```diff
@@ -362,6 +369,8 @@ class ProcessContext : public controller::ControllerServiceLookup, public core::
   controller::ControllerServiceProvider* controller_service_provider_;
   // state manager provider
   std::shared_ptr<core::CoreComponentStateManagerProvider> state_manager_provider_;
+  // state manager
+  std::shared_ptr<CoreComponentStateManager> stateManager_;
```

Comment:
Use snake_case_ here as well (valid for any further member variables in the PR), also I think these member comments are unnecessary as they do not contain any useful info.

Reply:
Done.